Lib/ufo2ft/__init__.py

```diff
@@ -1,7 +1,8 @@
-from kernFeatureWriter import KernFeatureWriter
-from makeotfParts import FeatureOTFCompiler
-from markFeatureWriter import MarkFeatureWriter
-from outlineOTF import OutlineOTFCompiler, OutlineTTFCompiler
+from __future__ import print_function, division, absolute_import
+from .kernFeatureWriter import KernFeatureWriter
```

Comment:
I'd actually like all the relative imports from ufo2ft to be spelled out explicitly e.g. `from ufo2ft.kernFeatureWriter ...`